vector/src/main/java/im/vector/riotx/features/home/room/detail/RoomDetailFragment.kt

```diff
@@ -618,8 +621,14 @@ class RoomDetailFragment :
 
     override fun onImageMessageClicked(messageImageContent: MessageImageContent, mediaData: ImageContentRenderer.Data, view: View) {
         // TODO Use navigator
-        val intent = ImageMediaViewerActivity.newIntent(vectorBaseActivity, mediaData)
-        startActivity(intent)
+
+        val intent = ImageMediaViewerActivity.newIntent(vectorBaseActivity, mediaData, ViewCompat.getTransitionName(view))
+        var bundle: Bundle? = null
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
```

Comment:
SDK version is already tested in `makeSceneTransitionAnimation()`, the code can be simplified

Reply:
done